core/Plugin/Dimension/ActionDimension.php

```diff
@@ -191,6 +191,23 @@ public function onNewAction(Request $request, Visitor $visitor, Action $action)
     }
 
     /**
+     * Returns `true` if the column for this dimension is a reference to the `log_action` table (ie, an "idaction column"),
+     * `false` if otherwise.
+     *
+     * @return bool
+     */
+    public function isActionReference()
```

Comment:
I can move it to DimensionMetadataProvider as a private method. It's not used anywhere else (I didn't do any refactoring).